### ISSUE TITLE
CVS bulk import of coordinates fails

### ISSUE THREAD

**Issue by thomasgi1** [mattieserver/netbox-topology-views#442]
### NetBox version

v3.6.5

### Topology Views version

v3.8.1

### Steps to Reproduce

1. Create some topology view with saved coordinates
2. Click on 'Device Coordinates'
3. Click on 'Export' button
4. Select 'All Data (CSV)'
5. Edit the CSV header line in saved 'netbox_coordinates.csv' file, so that all column headers become lower-case
6. Click on 'Import' button
7. Select 'Upload file' tab
8. Select the edited 'netbox_coordinates.csv' file as data file
9. Click on 'Submit' button

Two errors are shown:
Record 1 group: Select a valid choice. That choice is not one of the available choices.
Record 1 device: Select a valid choice. That choice is not one of the available choices.

### Expected Behavior

The device coordinates should be read.

### Observed Behavior

Two errors are shown:
Record 1 group: Select a valid choice. That choice is not one of the available choices.
Record 1 device: Select a valid choice. That choice is not one of the available choices.

Additionally there is no hint, what the available choices would be.

**Comment by dreng** [comment]
Hi,

as for every import form in NetBox, an export is not intended to be imported back into NetBox. Do not try to import it. It won't work.

I assume that you tried to import a CSV file that contains column names. NetBox expects pure data without column names.

> Additionally there is no hint, what the available choices would be.

This is not true actually. Just have a look at the field options at the lower end of the import dialog.

That said, I just tried to import a one-liner. I got no errors but the data does not show up in NetBox. That seems to be some kind of error.
